tests/e2e/data/services/test_isolines.py

```diff
@@ -116,18 +117,18 @@ def test_isochrones_from_dataframe_dataset(self):
         iso = Isolines(credentials=self.credentials)
 
         df = pd.DataFrame(self.points, columns=['name', 'the_geom'])
-        ds = Dataset(df, credentials=self.credentials)
+        cdf = CartoDataFrame(df, credentials=self.credentials)
```

Comment:
CartoDataFrame does not have a `credentials` param. Also, you can unify these two lines with the following, because a CDF is also a DF.

```
cdf = CartoDataFrame(self.points, columns=['name', 'the_geom'])
```